contributing.md

```diff
@@ -56,9 +56,9 @@ below. (Naively linking the binary is not sufficient to develop locally.)
 You can build the project, including all type definitions, with:
 
 ```bash
-yarn build
+pnpm run build
```

Comment:
I think `pnpm build` works too just like yarn

Reply:
Ah yeah can keep the shorthand here